plugin-attribution/src/main/java/com/mapbox/maps/plugin/attribution/AttributionViewImpl.kt

```diff
@@ -71,7 +71,12 @@ class AttributionViewImpl @JvmOverloads constructor(
    * @param bottom Margin to the bottom in pixel
    */
   override fun setAttributionMargins(left: Int, top: Int, right: Int, bottom: Int) {
-    (layoutParams as FrameLayout.LayoutParams).setMargins(left, top, right, bottom)
+    (layoutParams as FrameLayout.LayoutParams).apply{
+      marginStart = left
+      setTop(top)
```

Comment:
we might want to use `marginTop` instead of `setTop` as we are setting the attributionMargins in this function. same : `marginBottom` instead of `setBottom`